vamdctap/generators.py

```diff
@@ -536,13 +548,13 @@ def XsamsSources(Sources, tap):
 <Year>%s</Year>""" % ( G('SourceTitle'), G('SourceCategory'),
                        G('SourceYear') )
 
-        yield makeOptionalTag('SourceName','SourceName',G)
+        yield makeEscapedOptionalTag('SourceName','SourceName',G)
         yield makeOptionalTag('Volume','SourceVolume',G)
         yield makeOptionalTag('PageBegin','SourcePageBegin',G)
         yield makeOptionalTag('PageEnd','SourcePageEnd',G)
         yield makeOptionalTag('ArticleNumber','SourceArticleNumber',G)
-        yield makeOptionalTag('UniformResourceIdentifier','SourceURI',G)
-        yield makeOptionalTag('DigitalObjectIdentifier','SourceDOI',G)
+        yield makeEscapedOptionalTag('UniformResourceIdentifier','SourceURI',G)
```

Comment:
Isn't it confusing to call things "escaped" when they are not yet, and escaping still needs to be done? Maybe drop the "d" ?

Also, will this work for other nodes? e.g. when they have pre-escaped the SourceURI.